docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
Please use env to get the bash instead of targeting a specific binary.
```suggestion
#!/usr/bin/env bash
```